### PR TITLE
big batch of text changes. tweaks everywhere

### PR DESCRIPTION
**Pull request by jasonmcalpin**
Even more tweaks and copy edits cleaning up and matching text in pdf

### CODE REVIEW

**Comment by eepMoody** [comment]
+1

**Comment by jasonmcalpin** [comment]
i think we are a go.

File: source/classes/wizard.rst [hunk 1]
```diff
@@ -308,50 +308,51 @@ time you use this feature again before finishing a long rest, the
...
-to find new spells to do so, as normal. For this reason, many
-wizards keep backup spellbooks in a safe place.
-
-**The Book’s Appearance.** Your spellbook is a unique compilation
-of spells, with its own decorative flourishes and margin notes. It
-might be a plain, functional leather volume that you received as a
-gift from your master, a finely bound gilt-­‐edged tome you found in
-an ancient library, or even a loose collection of notes scrounged
-together after you lost your previous spellbook in a mishap.
+    .. rubric:: Your Spellbook
```
Comment (maxrothman) [inline source/classes/wizard.rst]: Why this change?

Comment (jasonmcalpin) [inline source/classes/wizard.rst]: they are added to all callouts from the pdf version.

**Comment by maxrothman** [comment]
:+1:
